WalletWasabi.Fluent/ViewModels/AddWalletPageViewModel.cs

```diff
@@ -46,6 +46,11 @@ public class AddWalletPageViewModel : NavBarItemViewModel
 			});
 
 			ImportWalletCommand = ReactiveCommand.Create(() => new ImportWalletViewModel(navigationState, WalletName, walletManager));
+			
+			ConnectHardwareWalletCommand = ReactiveCommand.CreateFromTask(async () =>
```

Comment:
If this is async, where is the await? If there is none just use ReactiveCommand.Create